### PR TITLE
Added gif_path as a configuration parameter in the admin GUI

### PR DESCRIPTION
**Pull request by fdurand**
This branch adds the possibility to set the path of the gif for the network detection.

Update:
## Description

This branch add gif_path:
Permit to change the path of the network detection gif
## Impacts

no
## Delete branch after merge

YES

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Look good to me just fix the merge conflict

**Comment by fdurand** [comment]
rebase ok

File: conf/pf.conf.defaults [hunk 1]
```diff
@@ -697,6 +697,12 @@ network_detection=enabled
 #
 network_detection_ip = 192.95.20.194
 #
+# captive_portal.gif_path
+#
+# This is the path where the gif is on the webserver to detect if the network access
+# has been enabled.
+gif_path = /common/network-access-detection.gif?r=
```
Comment (cgx) [inline conf/pf.conf.defaults]: Please append the `?r=` in JavaScript instead.

**Comment by cgx** [comment]
It doesn't have to be a GIF, therefore please rename the configuration parameter to `img_path`.

**Comment by extrafu** [comment]
Indeed, or simply image_path - we aren't trying to save bytes here :-)

**Pull request merged**